param/parameterized.py

```diff
@@ -979,6 +1085,65 @@ def inspect_value(self_,name): # pylint: disable-msg=E0213
 
         return value
 
+
+    def params_depended_on(self_,name):
```

Comment:
This and several more below need docstrings...

Reply:
I'm also not sure about the names yet but we can iterate on that outside this PR.

Reply:
I agree - will be part of an issue covering things that need to be sorted out before the next release.